src/PrestaShopBundle/Resources/views/Admin/Module/Includes/modal_confirm.html.twig

```diff
@@ -53,7 +53,11 @@
                     {{ "Your current settings will be saved, but the module will no longer be active."|trans({}, 'Admin.Modules.Notification') }}
                   {% endif %}
                   {% if module_action == 'uninstall' %}
-                    {{ "You are about to uninstall %moduleName% module."|trans({'%moduleName%': module.attributes.displayName}, 'Admin.Modules.Notification') }}
+                    {% if module.attributes.confirmUninstall %}
+                      {{ module.attributes.confirmUninstall }}
+                    {% else %}
+                      {{ "You are about to uninstall %moduleName% module."|trans({'%moduleName%': module.attributes.displayName}, 'Admin.Modules.Notification') }}
```

Comment:
ping @superGB.

What do you think of displaying this message at all times? With the optionnal confirmUninstall displayed after?